src/core/command/command-not-found.ts

```diff
@@ -0,0 +1,5 @@
+export class CommandNotFound extends Error {
```

Comment:
Suffix the class name with Error: `CommandNotFoundError` for organizational purposes.